freertos-rust/src/utils.rs

```diff
@@ -9,6 +9,7 @@ pub struct TypeSizeError {
     rust_size: usize,
 }
 
+#[cfg(feature = "cpu_clock")]
```

Comment:
Should this cfg line also be added to the definition of `freertos_rs_get_configCPU_CLOCK_HZ()` in `shim.rs`? I'm thinking that if this idea is expanded to other `#define`'d stuff, we should exclude on the lowest level possible.